routes/wishlist.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link rel="stylesheet" href="../styles/wishlist.css" />
+    <title>Sneakoholics | Login</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar fixed">
+        <a href="#" class="nav-heading">Sneakoholics</a>
+        <input
+          type="text"
+          class="input"
+          placeholder="What do you want to wear today?"
+        />
+        <ul class="nav-actions">
+          <li class="nav-action-item">
+            <button class="btn btn-primary">
+              <a href="./login.html">Login</a>
+            </button>
+          </li>
+          <li class="nav-action-item">
+            <div class="badge-container">
+              <i class="fas fa-shopping-cart fa-2x" aria-hidden="true"></i>
+              <span class="badge badge-icon">10</span>
+            </div>
+          </li>
+          <li class="nav-action-item">
+            <div class="badge-container">
+              <i class="far fa-heart fa-2x"></i>
+              <span class="badge badge-icon">10</span>
+            </div>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="h3 txt-center">My Wishlist</div>
+        <hr />
+        <div class="wishlist-container">
+          <div class="card card-ecom card-shadow">
+            <div class="card-body badge-container">
+              <div class="card-img">
+                <img
+                  src="../assets/images/sneaker.jpg"
+                  alt=""
```

Comment:
You should add alt-text for accessibility

Reply:
done